src/utils/hmac-validator.ts

```diff
@@ -36,8 +36,8 @@ export default function validateHmac(query: AuthQuery): boolean {
       'Query does not contain an HMAC value.',
     );
   }
-  const {hmac, ...rest} = query;
-  const localHmac = generateLocalHmac(rest);
+  const {hmac, code, timestamp, state, shop} = query;
+  const localHmac = generateLocalHmac({code, timestamp, state, shop});
```

Comment:
Should we include the `host` argument if it's here? I suspect this might fail if we're using App Bridge v2, but I could be wrong.

Reply:
Probably. I was working off of the types included in the lib because I forgot what values we were working with before. Will double check!

Reply:
Done.